bundles/org.openhab.binding.flicbutton/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="flicbutton"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<bridge-type id="flicd-bridge">
+		<label>FlicButton Bridge</label>
+		<description>This bridge represents a running instance of the fliclib-linux-hci server (flicd).</description>
+
+		<config-description>
+			<parameter name="hostname" type="text" required="true">
+				<context>network-address</context>
+				<label>flicd Hostname</label>
+				<description>IP or Host name of the Flic daemon (flicd).</description>
+				<default>localhost</default>
+			</parameter>
+			<parameter name="port" type="integer" required="true">
```

Comment:
When a default is provided, we usually say that required should be set to false, since if no setting is given, the default will be used.